mesa/experimental/jupyter_viz.py

```diff
@@ -60,11 +80,23 @@ def handle_change_model_params(name: str, value: any):
         set_model_parameters({**model_parameters, name: value})
 
     # 3. Set up UI
-    solara.Markdown(name)
-    UserInputs(user_params, on_change=handle_change_model_params)
-    ModelController(model, play_interval, current_step, set_current_step, reset_counter)
+    with solara.AppBarTitle():
```

Comment:
```suggestion
    solara.Title(name)
```

This also sets the tab title